src/modules/rlm_delay/rlm_delay.c

```diff
@@ -225,7 +225,7 @@ static xlat_action_t xlat_delay(TALLOC_CTX *ctx, UNUSED fr_cursor_t *out,
 {
 	rlm_delay_t const	*inst;
 	void			*instance;
-	struct timeval		delay_tv;
+	fr_time_delta_t		delay_at;
```

Comment:
It's not "delay at" that'd be delay at a wallclock time, this is a delta.  Just "delay" is fine.